zio-http/shared/src/main/scala/zio/http/Body.scala

```diff
@@ -650,4 +650,51 @@ object Body {
 
   private val zioEmptyChunk = Exit.succeed(Chunk.empty[Byte])
 
+  private[zio] final case class StringBody(
+    data: String,
+    charset: Charset = Charsets.Http,
+    override val contentType: Option[Body.ContentType] = None,
+  ) extends UnsafeBytes {
+
+    var bytes: Array[Byte] = null
```

Comment:
This should be private right?

Reply:
Yes

Reply:
I did this mainly, because this mimics the behavior of nettys AsciiString class, that keep the bytes cached.
If the body is used by us, we call it at least once.

More then once, maybe if we use known size. I will check. Maybe remove it

Reply:
@987Nabil Did you forget to make it private? 🤔

Reply:
@987Nabil if it's invoked at least once, does it make sense to store it as a String in the class to begin with? Can't we simply make the conversion to `Array[Byte]` and store it as such instead?

Reply:
It doesn't make a difference as long as we don't cache the `Array[Byte]` in a var, because in that case the class contains an extra field which increases memory pressure (especially for large responses).